app/src/main/java/org/simple/clinic/enterotp/EnterOtpScreenController.kt

```diff
@@ -97,4 +98,33 @@ class EnterOtpScreenController @Inject constructor(private val userSession: User
         .compose(NewlyVerifiedUser())
         .map { { ui: Ui -> ui.goBack() } }
   }
+
+  private fun resendSms(events: Observable<UiEvent>): Observable<UiChange> {
+
```

Comment:
Empty line. Static analysis should catch this :(

Reply:
Fixed.